server/README.md

```diff
@@ -5,3 +5,11 @@
 ## Running the Server
 
 > node index.js
+
```

Comment:
This is an internal implementation detail, so there's no need to add this to the external README. Please delete this.